app/code/Magento/CatalogGraphQl/Model/Resolver/Products/Query/Search.php

```diff
@@ -148,4 +160,27 @@ private function buildSearchCriteria(array $args, ResolveInfo $info): SearchCrit
 
         return $searchCriteria;
     }
+
+    /**
+     * Get product attribute value
+     *
+     * @param Product $product
+     * @param array $productArray
+     * @param string $field
+     *
+     * @return string|null
+     */
+    private function getAttributeValue(Product $product, array $productArray, string $field)
```

Comment:
I would suggest specifying the return type 
```suggestion
    private function getAttributeValue(Product $product, array $productArray, string $field): ?string
```

Reply:
**Done**

Reply:
Hi @rogyar 
I can't apply this change because attribute value can hava `int` and `bool ` values  
I provide this test as example
<img width="1440" alt="Screenshot 2020-10-25 at 01 06 52" src="https://user-images.githubusercontent.com/13517181/97185010-3b31dc00-17a8-11eb-8c5c-03ad2586570f.png">

Thanks